src/test/java/org/broadinstitute/hellbender/tools/walkers/vqsr/scalable/ScoreVariantAnnotationsIntegrationTest.java

```diff
@@ -229,6 +229,36 @@ public void testNoVariantsInInput() {
         Assert.assertTrue(new File(outputPrefix + ".vcf.idx").exists());
     }
 
+    /**
+     * If no variants of a given type are present in the input in the specified region,
+     * test that we do not attempt to create the temporary scores or annotations HDF5 files for that type.
```

Comment:
where does this check occur?

Reply:
A separate check does not occur in the production code, this test is itself a check. Sorry if the (hastily written) comment is a bit unclear!